### PR TITLE
New upstream version 4.4.2

### PR DESCRIPTION
**Pull request by knackebrot**

### CODE REVIEW

**Comment by nyanmisaka** [comment]
Can you separate it into two commits?
1. ffmpeg version bump
2. patches and build scripts update
`debian/patches`, `debian/control`, `debian/changelog`,
`build.yaml`, `build-win64:ffrevison`

**Comment by knackebrot** [comment]
Done.

File: debian/patches/0024-add-sub2video-option-to-subtitles-filter.patch [hunk 1]
```diff
@@ -1,8 +1,6 @@
-Index: jellyfin-ffmpeg/libavfilter/vf_subtitles.c
-===================================================================
```
Comment (nyanmisaka) [inline debian/patches/0024-add-sub2video-option-to-subtitles-filter.patch:2]: LTGM. But any reason why indexes are stripped?

File: debian/patches/0030-add-fixes-for-a-vaapi-qsv-mapping-error.patch [hunk 2]
```diff
@@ -1,8 +1,6 @@
-Index: jellyfin-ffmpeg/libavcodec/qsvenc_hevc.c
-===================================================================
```
Comment (nyanmisaka) [inline debian/patches/0030-add-fixes-for-a-vaapi-qsv-mapping-error.patch:2]: Same.

File: debian/patches/0030-add-fixes-for-a-vaapi-qsv-mapping-error.patch [hunk 3]
```diff
@@ -14,23 +12,22 @@ Index: jellyfin-ffmpeg/libavcodec/qsvenc_hevc.c
      { "qmin",      "-1"    },
      { "qmax",      "-1"    },
      { "trellis",   "-1"    },
-Index: jellyfin-ffmpeg/libavcodec/vaapi_decode.c
-===================================================================
```
Comment (nyanmisaka) [inline debian/patches/0030-add-fixes-for-a-vaapi-qsv-mapping-error.patch:18]: Same.

File: debian/patches/0024-add-sub2video-option-to-subtitles-filter.patch (hunk 1 above)
Comment (knackebrot) [reply debian/patches/0024-add-sub2video-option-to-subtitles-filter.patch:2]: I've finally started using quilt, and just pasted the .quiltrc from here: https://wiki.debian.org/UsingQuilt
I can redo it without QUILT_NO_DIFF_INDEX=1

Comment (nyanmisaka) [reply debian/patches/0024-add-sub2video-option-to-subtitles-filter.patch:2]: Thanks. I'll merge this first so as you can add `fdc_aac` to the debian changelog in another PR.

**Comment by knackebrot** [comment]
Hi @nyanmisaka, the patches were refreshed.

**Review by nyanmisaka — APPROVED** [review]

**Pull request merged**